### PR TITLE
Added the DevCon and Summit to the events

### PR DESCRIPTION
**Pull request by pedrominatel**
This PR includes the upcoming events for 2024.

- DevCon24
- Espressif Summit Brazil

### CODE REVIEW

File: content/events/summit-brazil-24/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+date: 2024-06-10
+draft: false
+description: "Espressif Systems Summit in Brazil"
+tags: ["event", "announcement"]
+showDate : false
+---
+
+## Espressif Summit Brazil 2024
+
+We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm. On the first day, we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:12]: Suggest changing the date and time to improve readability, maybe like this:

from July 17 to 18, 9 am - 6 pm.

We have a [chapter](http://mos.espressif.com/time-and-dates.html) in our Espressif Manual of Style about it, but the guidelines are quite flexible.

File: content/events/summit-brazil-24/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+On the second day, workshops will be presented in two different tracks, including "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32." All the talks and workshops will be presented by Espressif experts and will be mostly in **Portuguese**.
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:14]: The paragraphs combine unrelated things. Suggest splitting information among the paragraphs like this:

We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm.

The activities are as follows:
  - Day 1: we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
  - Day 2: workshops will be presented in two different tracks, including "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32."

All the talks and workshops will be presented by Espressif experts and will be mostly in **Portuguese**.

Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:14]: Two different tracks are announced. Among "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32," what is track 1 and what is track 2?

File: content/events/summit-brazil-24/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Location
+
+Vitoria Hotel Concept, Campinas, SP
+
+Time: Starting from 9 AM
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:20]: The time format above is "am". Suggest using time format consistently, either AM or am everywhere.

File: content/events/summit-brazil-24/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Espressif Summit Brazil 2024
+
+We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm. On the first day, we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
+
+On the second day, workshops will be presented in two different tracks, including "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32." All the talks and workshops will be presented by Espressif experts and will be mostly in **Portuguese**.
+
+### Location
+
+Vitoria Hotel Concept, Campinas, SP
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:18]: When I read this text I wonder where this location is. Most likely I will copy and paste it into Google to get more information. Should we add a link for this place ourselves: either the website of the conference hall or maybe Google Maps?

File: content/events/summit-brazil-24/index.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+| 15:30 | Desenvolvimento Arduino como componente do ESP-IDF | Rodrigo Garcia - Software Engineer |
+| 16:00 | Boas práticas de rede e MQTT com o ESP32 | Euripedes Rocha - Networking Manager |
+| 17:00 | Introdução ao Zephyr e debugging com openOCD | Lucas Tamborrino - Software Engineer |
+| 18:00 | Q&A geral e networking | Espressif Systems |
+
+## Registration
+
+The event is free for any participant, and registration can be done for each day and workshop with the following links:
+
+- Talks on July 17th: [Espressif Summit Brazil 2024](https://espressif.us15.list-manage.com/track/click?u=40830afd8eb6f70ab5e47b7a4&id=1988e2d518&e=e3a0757cf1)
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:47]: Here and below, suggest using the date format recommended above for better readability: July 17.

File: content/events/summit-brazil-24/index.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Workshops July 18th:
+
+- Workshop: [Introdução ao ESP-IDF com o ESP32-C6 (EN) - 10:00 – 13:00](https://espressif.us15.list-manage.com/track/click?u=40830afd8eb6f70ab5e47b7a4&id=befed46386&e=e3a0757cf1)
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:51]: Suggest using the time format consistently: either 24-hour format or am/pm everywhere.

File: content/events/devcon24/index.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+showDate : false
+---
+
+{{< badge >}}
+New event!
+{{< /badge >}}
+
+## DevCon24
+
+Espressif DevCon24, the third edition of Developers conference made by Developers for Developers!
```
Comment (f-hollow) [inline content/events/devcon24/index.md:16]: Suggest:

```suggestion
After [DevCon22](link to more details) and [DevCon23](link to more details), Espressif DevCon24 is the third developer conference made by developers for developers!
```

File: content/events/devcon24/index.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+{{< alert icon="eye">}}
+**The DevCon24, will take place online from September 3rd to 5th 13.00 to 18.00 CEST.**
```
Comment (f-hollow) [inline content/events/devcon24/index.md:19]: Suggest:

from September 3 to 5, 13.00 - 18.00 CEST

The way how we format dates and time should be consistent. See also my suggestions for the other event.

File: content/events/devcon24/index.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+{{< /alert >}}
+
+During the three-day online conference, we will provide you with nearly 30 talks created by Espressif technical experts and its partners, covering a wide range of topics, including Thread, Matter, ESP-IDF, Arduino Core for ESP32, ESP-Mesh-Lite + ESP RainMaker Cloud solution, RUST and more.
```
Comment (f-hollow) [inline content/events/devcon24/index.md:22]: Not clear what `its partners` refers to. Maybe company partners or community members?

File: content/events/devcon24/index.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+> *Full program will be shared later.*
+
+For more information, please visit [DevCon24](devcon.espressif.com)
+
+Event is for free and will be streamed on [Espressif's YouTube channel](https://www.youtube.com/channel/UCDBWNF7CJ2U5eLGT7o3rKog).
```
Comment (f-hollow) [inline content/events/devcon24/index.md:28]:
```suggestion
This event is free and will be streamed on [Espressif's YouTube channel](https://www.youtube.com/channel/UCDBWNF7CJ2U5eLGT7o3rKog).
```

File: content/events/devcon24/index.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+You can watch here, the DevCon23 keynote from our CEO, Teo Swee Ann.
```
Comment (f-hollow) [inline content/events/devcon24/index.md:30]:
```suggestion
Here is the DevCon23 keynote from our CEO, Teo Swee Ann:
```

File: content/events/devcon24/index.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+{{< youtube mR3gUNXMEsM >}}
+
+---
+
+## About Espressif Systems
+
+Espressif Systems focuses on the R&D and production of wireless communication chips and modules and the development of operating system software. Established in 2008 with headquarters in Shanghai, the company has offices in Brazil, the Czech Republic, India and Singapore.
```
Comment (f-hollow) [inline content/events/devcon24/index.md:38]: Is it a standard description? The text can be improved. Too many "and" here. What is meant by operating system software?

**Review by f-hollow — COMMENTED** [review]
A few nitpicks to make formatting consistent and improve some wording here and there.

File: content/events/devcon24/index.md (hunk 12 above)
Comment (pedrominatel) [reply content/events/devcon24/index.md:38]: I'll remove it. This is from the LinkedIn event post.

**Comment by pedrominatel** [comment]
@f-hollow, thanks for the review. Can you please take a look and review the DevCon22 and DevCon23 posts?

File: content/events/summit-brazil-24/index.md (hunk 2 above)
Comment (f-hollow) [reply content/events/summit-brazil-24/index.md]: > Day 2: workshops will be presented in two different tracks, including “Introduction to ESP-IDF with the ESP32-C6,” “IoT Workshop with RainMaker and Matter,” and “Zephyr with ESP32.”

It is still unclear how the tracks are related to the listed topics. Please see if it is important to clarify. If it is not, feel free to resolve this discussion point.

**Comment by f-hollow** [comment]
@pedrominatel Left a question about an existing comment and suggested to update one heading. Also, the Brazil event still has ac couple of dates in the format "July 17th", suggest using July 17.

 Otherwise, LGTM!

**Comment by f-hollow** [comment]
@pedrominatel Left a question about an existing comment and suggested to update one heading. Also, the Brazil event still has ac couple of dates in the format "July 17th", suggest using July 17.

Otherwise, LGTM!

File: content/events/summit-brazil-24/index.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+
+### Location
+
+[Vitoria Hotel Concept, Campinas, SP](https://www.google.com/travel/hotels/s/L7pJr99Wecv6bma46)
+
+Time: Starting from 9 am
+
+Use the promotional code ESPRESSIF and get a 12% discount on your stay.
+
+## Agenda
```
Comment (f-hollow) [inline content/events/summit-brazil-24/index.md:29]:
```suggestion
## Agenda for July 17
```

**Comment by f-hollow** [comment]
@pedrominatel Left a question about an existing comment and suggested to update one heading. Also, the Brazil event still has ac couple of dates in the format "July 17th", suggest using July 17.

Otherwise, LGTM!

**Review by f-hollow — APPROVED** [review]

**Pull request merged**